client/src/pages/email-sign-up.tsx

```diff
@@ -61,7 +61,7 @@ function EmailListOptIn({
   const { t } = useTranslation();
   if (isSignedIn) {
     return (
-      <Row>
+      <Row className='email-list-opt'>
```

Comment:
Hi @huyenltnguyen ,
 I understand your concern about avoiding the usage of CSS overrides in the code. I have another approach. Instead of modifying the CSS, we can add the "flex" class to the row component like this: ```<Row className="flex">```.

This will result in a slight difference in the layout in mobile view. The two buttons will appear on the same line instead of being stacked vertically.

Do you think this alternative solution could be helpful?

Reply:
+ I have identified one more issue along with this. Due to the ```Row``` component which has margin of ```-15px``` by default, it is causing an empty space and as a result footer is not properly aligned in mobile view.
![image](https://github.com/freeCodeCamp/freeCodeCamp/assets/64531160/c07d93e1-e4b6-409c-98e5-7f33bdc43bac)


So alternate suggestion is to add  ```<Container className='flex'>```  instead  ```Row```, which will solve both button alignment and extra space in footer without CSS. 

![image](https://github.com/freeCodeCamp/freeCodeCamp/assets/64531160/8a22f35c-6976-404e-ae4c-d7ccedc91626)

Would this approach be acceptable in your view?